cmake/CeleritasConfig.cmake.in

```diff
@@ -43,10 +43,26 @@ endif()
 include(CMakeFindDependencyMacro)
 
 if(CELERITAS_USE_CUDA)
-  enable_language(CUDA)
   find_dependency(CUDAToolkit REQUIRED QUIET)
+
+  if(CELERITAS_USE_VecGeom AND NOT CMAKE_CUDA_COMPILER_LOADED)
+    if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+      enable_language(CUDA)
```

Comment:
It looks like the ORANGE ndebug (static, no vecgeom) build fails CI: so presumably something in CMake requires the language to be enabled downstream if an upstream static build uses it. This makes sense I think.